apiserver/pkg/model/converter.go

```diff
@@ -74,6 +114,31 @@ func PopulateHeadNodeSpec(spec v1alpha1.HeadGroupSpec) *api.HeadGroupSpec {
 		ComputeTemplate: spec.Template.Annotations[util.RayClusterComputeTemplateAnnotationKey],
 	}
 
+	for _, annotation := range getNodeDefaultAnnotations() {
+		delete(spec.Template.Annotations, annotation)
+	}
+	if len(spec.Template.Annotations) > 0 {
+		headNodeSpec.Annotations = spec.Template.Annotations
+	}
+
+	for _, label := range getNodeDefaultLabels() {
```

Comment:
I've seen some overlap of the default annotation and labels between node and ray cluster, here the reason to delete them is?